op-challenger/metrics/metrics.go

```diff
@@ -20,6 +20,11 @@ type Metricer interface {
 
 	// Record Tx metrics
 	txmetrics.TxMetricer
+
+	ClearTrackedGames()
+	RecordGameInProgress()
+	RecordGameDefenderWon()
+	RecordGameChallengerWon()
```

Comment:
To make this atomic I think it will need to be something more like `RecordTrackedGames(inProgressCount int, defenderWon int, challengerWon int)` or possibly passing a `map[GameStatus]int` makes sense but then you need to ensure you treat anything not present as 0 so it doesn't fail to update when a particular game status isn't present.